tests/aws/services/cloudwatch/test_cloudwatch.py

```diff
@@ -1391,8 +1391,17 @@ def contains_receive_delete_metrics() -> int:
 
     @markers.aws.validated
     @markers.snapshot.skip_snapshot_verify(
-        paths=["$..DashboardArn", "$..DashboardEntries..Size"], condition=is_old_provider
-    )  # ARN has a typo in moto
+        condition=is_old_provider,
+        paths=[
+            "$..DashboardArn",  # ARN has a typo in moto
+            "$..DashboardEntries..Size",
```

Comment:
NIT: we shouldn't need the `"$..DashboardEntries..Size"` here, as the other skip is unconditional and therefore includes the skip already. But nothing urgent.

Reply:
nice catch 🙌 have fixed this